### PR TITLE
updating dependencies for Terraform v13

### PR DESCRIPTION
**Pull request by klehmer**
This PR updates the necessary dependencies to use Terraform v13. This is a breaking change for existing modules and requires some manual state manipulation documented [here](https://www.notion.so/fishtownanalytics/Terraform-13-and-EKS-Module-Count-Upgrades-3331692eacaf40e984d2d77d540d0d45). The process described in the Notion page has been tested successfully twice in the staging environment.

This PR also adds `.DS_STORE` to the .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@
 # generated dbt config files
 dbt_config.sh
 config.yaml
+
+.DS_STORE
```
Comment (brian-fa) [inline .gitignore:15]:
```suggestion
.DS_Store
```

File: README.md [hunk 2]
```diff
@@ -83,19 +83,19 @@ Note that the `kubectl kots install` will prompt the user for a password for the
 |------|---------|
 | hashicorp/aws | >= 2.0 |
 | hashicorp/random | >= 3.0.0 |
-| hashicorp/kubernetes | >= 1.13.0 |
+| hashicorp/kubernetes | >= 1.9 |
```
Comment (brian-fa) [inline README.md:86]: Just a note to make sure you meant to go from `>= 1.13.0` to `>= 1.9` here.

**Review by brian-fa — CHANGES_REQUESTED** [review]
Just curious, what version number do you plan to use for this release?

File: .gitignore (hunk 1 above)
Comment (klehmer) [reply .gitignore:15]: Good catch!

File: README.md (hunk 2 above)
Comment (klehmer) [reply README.md:86]: Yes. I realized that I had manually entered the wrong version here (we actually define 1.9 or above in the K8s provider.

**Comment by klehmer** [comment]
@brian-fa I'm going to write a post about this in Slack but I'm thinking of incrementing the minor digit to 0.2.0 as this is a breaking change. Will post soon!

**Review by brian-fa — APPROVED** [review]

**Pull request merged**